addon/-private/system/relationships/belongs-to.js

```diff
@@ -85,8 +85,8 @@ import { DEBUG } from '@glimmer/env';
   ```
 
   In contrast to async relationship, accessing a sync relationship
-  will never trigger a request to fetch the resource,
-  and it will always return the existing local resource.
+  will always return the record (Model instance) for the existing
```

Comment:
I think we should say something about the fact that we error here if you access the relationship prior to the related resources being available (see my original feedback for suggested text).